converter/ena/base.py

```diff
@@ -9,7 +9,10 @@ def __init__(self, ena_type: str, xml_spec: dict):
         self.ena_type = ena_type
         self.xml_spec = xml_spec
 
-    def convert(self, entity: dict, additional_attributes: dict = {}):
```

Comment:
What was wrong with how it was before?

Reply:
@jacobwindsor I found this out in the painful way:
"In Python, when passing a mutable value as a default argument in a function, the default argument is mutated anytime that value is mutated.

Here, "mutable value" refers to anything such as a list, a dictionary or even a class instance."

https://florimond.dev/en/posts/2018/08/python-mutable-defaults-are-the-source-of-all-evil/

https://stackoverflow.com/questions/26320899/why-is-the-empty-dictionary-a-dangerous-default-value-in-python